README.md

```diff
@@ -7,6 +7,8 @@ Register [Marathon](https://mesosphere.github.io/marathon/) Tasks as [Consul](ht
 forwards it to Consul agents. It also re-syncs all the information from Marathon 
 to Consul on startup and repeats it with given interval.
 
+![marathon-consul](https://cloud.githubusercontent.com/assets/1616386/13946637/0f2ca0fe-f016-11e5-9711-75bfcc1b5f3c.gif)
```

Comment:
i think we should put differences somewhere in front of this. and this gif tells me nothing about how this service works - it's too fast ,too little information